catalyst_voices/packages/libs/catalyst_key_derivation/Earthfile

```diff
@@ -0,0 +1,23 @@
+VERSION 0.8
+
+IMPORT github.com/input-output-hk/catalyst-ci/earthly/flutter_rust_bridge:v3.2.24 AS flutter_rust_bridge
+
+builder:
+    FROM flutter_rust_bridge+builder
+    COPY . .
+
+# Generated necessary files for running Flutter web locally and save it locally.
+code-generator:
```

Comment:
I would rather split this target into two `code-generator` and `code-generator-local` which would be more convenient to run just `code-generator-local` from command line.

Though I would make it like this
```Earthfile
code-generator-base:
    FROM +builder
    DO flutter_rust_bridge+CODE_GENERATOR_WEB
    
code-generator: 
    FROM +code-generator-base
    
   SAVE ARTIFACT ./assets/js assets_js
   SAVE ARTIFACT ./rust/src/frb_generated.rs frb_generated.rs
   SAVE ARTIFACT ./lib/src src

code-generator-local: 
    FROM +code-generator-base
    
    SAVE ARTIFACT ./assets/js AS LOCAL ./assets/js
    SAVE ARTIFACT ./rust/src/frb_generated.rs AS LOCAL ./rust/src/frb_generated.rs
    SAVE ARTIFACT ./lib/src AS LOCAL ./lib/src
```